layers/gpu/spirv/bindless_descriptor_pass.cpp

```diff
@@ -78,7 +78,7 @@ uint32_t BindlessDescriptorPass::FindTypeByteSize(uint32_t type_id, uint32_t mat
 // Find outermost buffer type and its access chain index.
 // Because access chains indexes can be runtime values, we need to build arithmetic logic in the SPIR-V to get the runtime value of
 // the indexing
-uint32_t BindlessDescriptorPass::GetLastByte(BasicBlock& block) {
+uint32_t BindlessDescriptorPass::GetLastByte(BasicBlock& block, InstructionIt* inst_it) {
```

Comment:
```suggestion
uint32_t BindlessDescriptorPass::GetLastByte(BasicBlock& block, InstructionIt& inst_it) {
```
Unless it can sometimes be null